debian/snapd.install

```diff
@@ -1,4 +1,5 @@
 /usr/bin/snap
+/usr/bin/snapctl
```

Comment:
I wonder if we want this in PATH or rather hide it a bit more into e.g. `/usr/lib/snapd/` ?